alerting/src/test/java/org/hawkular/metrics/alerting/PublishDataPointsTest.java

```diff
@@ -62,7 +61,7 @@
 import rx.Observable;
 
 /**
- * TODO: We need to mock out AlertsService, I think, so that InsertedDataSubscriber can instantiate correctly
+ * Note that this test no longer tests filtering, as data is now filtered in alerting, not metrics.
```

Comment:
Then, perhaps this test is purely outdated.
Main goal was to test the bus reception of datapoints and next we extended to filtering.
So, I guess this could be removed and added new ones (in a following PR) to test a whole lifecycle.
In Hawkular Services there are a couple of similar tests that can be re-used (I think).